### ISSUE TITLE
Quiet dropping of days when slurry mass drops at end

### ISSUE THREAD

**Issue by sashahafner** [AU-BCE-EE/ABM#45]
With this input for slurry mass

![image](https://github.com/AU-BCE-EE/ABM/assets/35272876/8124dc96-1a19-4ada-b767-b390890ca90b)

the output ends at day 278.

76ac0a492e14ce7f735da865ba9bb9993032121f

**Comment by fdalby** [comment]
@sashahafner yes, so far I have added an artificial endpoint with minimally more slurry mass, which has been annoying. It might be related to #30 ?

**Comment by fdalby** [comment]
added a warning when this occurs.

**Comment by sashahafner** [comment]
This is related to interpretation of emptying/removal as instant, discussed in issue #30 .
